pkg/cli/build/build.go

```diff
@@ -199,6 +201,31 @@ func isImageDefinitionValid(ctx *image.Context) bool {
 	return true
 }
 
+func appendElementalRPMs(ctx *image.Context) {
+	elementalDir := filepath.Join(ctx.ImageConfigDir, "elemental")
+	if _, err := os.Stat(elementalDir); err != nil {
+		if !errors.Is(err, fs.ErrNotExist) {
+			zap.S().Warnf("Looking for 'elemental' dir failed unexpectedly: %s", err)
+		}
+
+		return
+	}
+
+	audit.Audit("Elemental registration is configured. The necessary RPM packages will be downloaded.")
+	zap.S().Info("Elemental registration requested")
+
+	packageList := ctx.ImageDefinition.OperatingSystem.Packages.PKGList
+	packageList = append(packageList, "elemental-register", "elemental-system-agent")
+
+	// TODO: Use an official repository URL once it's out
+	repositories := ctx.ImageDefinition.OperatingSystem.Packages.AdditionalRepos
+	repositories = append(repositories,
+		image.AddRepo{URL: "https://download.opensuse.org/repositories/isv:/Rancher:/Elemental:/Staging/standard/"})
```

Comment:
Not to keep beating the same drum, but in my head, all of the logic/details around a specific component should be located in a singular location. How would you feel about having this be a constant in elemental.go so that it's the sole owner of all of this sort of elemental-specific data? 

We'll have to readdress when we get a less in-code mechanism for storing all of these values, but I think that further enforces that point. We'll be able to look in each component module and know all of the values we may want to pull out and make overridable.